projects/observability/src/shared/components/cartesian/d3/legend/cartesian-legend.ts

```diff
@@ -41,7 +41,8 @@ export class CartesianLegend<TData> {
     private readonly summaries: Summary[] = []
   ) {
     this.isGrouped =
-      this.series.length > 0 && this.series.every(seriesEntry => seriesEntry.groupName !== seriesEntry.name);
+      this.series.length > 0 &&
+      this.series.every(seriesEntry => !isEmpty(seriesEntry.groupName) && seriesEntry.groupName !== seriesEntry.name);
```

Comment:
we're getting empty group name and series names?

Reply:
I will take a look more into it, it is having a problem in app-dev

Reply:
in events page

Reply:
Ah crap, that's in staging too, we're going to need to rush an upgrade. How'd we miss this, this is the only spot this feature is used!